src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs

```diff
@@ -289,7 +289,19 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<IMethodologyFileRepository, MethodologyFileRepository>();
             services.AddTransient<IMethodologyImageService, MethodologyImageService>();
             services.AddTransient<IMethodologyAmendmentService, MethodologyAmendmentService>();
-            services.AddTransient<IMethodologyApprovalService, MethodologyApprovalService>();
+            services.AddTransient<IMethodologyApprovalService, MethodologyApprovalService>(provider =>
```

Comment:
Oh momma - all to wire in a specific BlobStorageService :smile:  Be nice to see this go!